jest-playwright.config.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  // https://github.com/playwright-community/jest-playwright#start-a-server
+  serverOptions: [
+    {
+      command: 'npm run bundle && npx serve -C -l 8000 dist/sdk',
+      port: 8000,
+      launchTimeout: 15000,
+    },
+    {
+      command: 'npx serve -l 9000 test/fixtures',
+      port: 9000,
+      launchTimeout: 25000,
+    },
```

Comment:
Do you think tidier to have two servers??

Reply:
I don't know. I guess we can have one serving the directory with the mock html and the dist artifacts together, but we will have to point the microbundle output to it.
I thought it could be a problem to have separated servers if there is a race condition between the two and somehow the bundle could not be served to the mock page, but it didn't happen with me until now.

Reply:
its fine like this I think.